understanding/intro.html

```diff
@@ -79,30 +79,30 @@ <h2>Understanding the Four Principles of Accessibility</h2>
-				<p>All WCAG 2.1 Success Criteria are written as testable criteria for objectively determining if content satisfies the Success Criteria. While some of the testing can be automated using software evaluation programs, others require human testers for part or all of the test.</p>
-				<p>Although content may satisfy the Success Criteria, the content may not always be usable by people with a wide variety of disabilities. Professional reviews utilizing recognized qualitative heuristics are important in achieving accessibility for some audiences. In addition, usability testing is recommended. Usability testing aims to determine how well people can use the content for its intended purpose.</p>
+				<p>Under each guideline, there are success criteria that describe specifically what must be achieved in order to <a class="gl-ref" href="https://www.w3.org/TR/WCAG/#dfn-conform">conform</a> to this standard. They are similar to the "checkpoints" in WCAG 1.0. Each success criterion is written as a statement that will be either true or false when specific Web content is tested against it. The success criteria are written to be technology neutral.</p>
+				<p>All WCAG 2 success criteria are written as testable criteria for objectively determining if content satisfies the success criteria. While some of the testing can be automated using software evaluation programs, others require human testers for part or all of the test.</p>
+				<p>Although content may satisfy the success criteria, the content may not always be usable by people with a wide variety of disabilities. Professional reviews utilizing recognized qualitative heuristics are important in achieving accessibility for some audiences. In addition, usability testing is recommended. Usability testing aims to determine how well people can use the content for its intended purpose.</p>
 				<p>The content should be tested by those who understand how people with different types of disabilities use the Web. It is recommended that users with disabilities be included in test groups when performing human testing. </p>
-				<p>Each Success Criterion for a guideline has a link to the section of the How to Meet document that provides:</p>
+				<p>Each success criterion for a guideline has a link to the section of the How to Meet document that provides:</p>
 				<ul>
 					<li>
-						<p>sufficient techniques for meeting the Success Criterion,</p>
+						<p>sufficient techniques for meeting the success criterion,</p>
 					</li>
 					<li>
 						<p>optional advisory techniques, and</p>
 					</li>
 					<li>
-						<p>descriptions of the intent of the Success Criteria, including benefits, and examples.</p>
+						<p>descriptions of the intent of the success criteria, including benefits, and examples.</p>
```

Comment:
```suggestion
						<p>descriptions of the intent of the success criterion, including benefits, and examples.</p>
```

Recommend making 'success criteria' singular in this instance, as it's referring to info on a single criterion.